### PR TITLE
[14.0][FIX] report_qweb_encrypt: conflict with bank statement

### PR DESCRIPTION
**Pull request by ps-tubtim**
This pull request is used to fix an error when **Validate** a bank statement.

### Step to error:

1. Install account and report_qweb_encrypt
2. Create and reconcile a bank statement
3. **Validate** a bank statement

Selection_082

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The PR is changing the signature, so it's not valid for a stable version, and moreover, there's a chance of singleton error in case more than one record is called for render.

**Comment by ps-tubtim** [comment]
@pedrobaeza I change the solution to the problem. Could you review it again?

File: report_qweb_encrypt/models/ir_actions_report.py [hunk 1]
```diff
@@ -34,6 +34,8 @@ def _render_qweb_pdf(self, res_ids=None, data=None):
         document, ttype = super(IrActionsReport, self)._render_qweb_pdf(
             res_ids=res_ids, data=data
         )
+        if not isinstance(res_ids, list):
```
Comment (pedrobaeza) [inline report_qweb_encrypt/models/ir_actions_report.py]:
```suggestion
        if isinstance(res_ids, int):
```

**Review by pedrobaeza — COMMENTED** [review]
I'm still not sure the problem should be fixed enough. Shouldn't the other module always pass a list?

File: report_qweb_encrypt/models/ir_actions_report.py (hunk 1 above)
Comment (ps-tubtim) [reply report_qweb_encrypt/models/ir_actions_report.py]: I have already updated this code.

**Comment by pedrobaeza** [comment]
But what about my question?

**Comment by ps-tubtim** [comment]
Most of the other modules passed values in a list, but I just found out that the bank statement is passed values in the integer.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**